dashboard/lib/dynamic_config/loaders/gatekeeper_loader.rb

```diff
@@ -0,0 +1,62 @@
+# A script to update the Gatekeeper settings
```

Comment:
Could you mention that the settings are loaded from a file and give an example of the file format?